test/relations.test.js

```diff
@@ -191,16 +202,21 @@ describe('relations', function() {
           .then(function(ch) {
             should.exist(ch);
             ch.should.have.lengthOf(3);
-
             var chapters = book.chapters();
             chapters.should.eql(ch);
-
-            return book.chapters.getAsync({order: 'name DESC'})
+            var nameFilter = isCassandraConnector ? {} : {order: 'name DESC'};
+            return book.chapters.getAsync(nameFilter)
```

Comment:
Ditto - remove the filter completely and use assertions that don't assume any order of the returned books.